### DISCUSSION TITLE
use react-query-devtools v4.0.10 publish

### DISCUSSION THREAD

**Discussion by ymseoooooo** [TanStack/query#4045] [Q&A] [answered]
I am trying to use `react-query` and `react-query-devtools` in next.js app built with yarn berry.
The versions I have installed are.

![image](https://user-images.githubusercontent.com/23565781/185034160-81645964-4f43-4b31-bb22-c746c2f105b1.png)

Yarn berry blocks all unintentional hoist.
So I recently saw someone having problems with `react-query` dependency in `react-query-devtools`.

It seems that the `react-query` peer dependency in the `react-query-devtools` package related to yarn berry has been added in version 4.0.10.

![image](https://user-images.githubusercontent.com/23565781/185027493-90032586-7507-4e17-be54-245b96a7dc70.png)

https://github.com/TanStack/query/pull/4000

However, if you actually install the `react-query-devtools` package in the terminal, the peer dependency is not added.

![image](https://user-images.githubusercontent.com/23565781/185028516-46a6fcad-1559-4072-849f-b4109063dac1.png)
https://www.npmjs.com/package/@tanstack/react-query-devtools

The bug was fixed 5 days ago, What I checked is that version 4.0.10 was published a month ago and has not been updated.

The `package.json` file of `react-query-devtools` that I installed in the terminal is as follows.
I installed the package via `yarn add @tanstack/react-query-devtools` in terminal.
also `yarn add @tanstack/react-query`
I did not select a specific version to install.
![image](https://user-images.githubusercontent.com/23565781/185031288-1a950ac5-f0b9-4a8b-8b22-f24c18d9f831.png)

The source I was thinking of is this.
![image](https://user-images.githubusercontent.com/23565781/185031544-f195459f-bf27-473d-a5fa-17fdfb08c9c3.png)
https://github.com/TanStack/query/blob/main/packages/react-query-devtools/package.json

What version do I need to install to use the version that fixes the peer dependency bug?

**Comment by TkDodo** [comment, marked as answer]
the release script is buggy, and versions haven't been released. I'll trigger a release of all packages now.
